### ISSUE TITLE
Using labels, colors, and other options

### ISSUE THREAD

**Issue by NathanWalker** [NathanWalker/nativescript-loading-indicator#1]
Great plugin, thanks for this!

Curious if the api needs to be expanded to include ways to utilize the other options as seen here:
https://raw.githubusercontent.com/jdg/MBProgressHUD/master/Demo/Classes/HudDemoViewController.m

Or would the `show()` method return an instance of `MBProgressHUD` that can be configured via javascript, for example:

```
var hud = loader.show(); 
hud.dimBackground = true;
hud.mode = MBProgressHUDModeText;
hud.labelText = @"Some message...";
hud.margin = 10;
hud.color = some javascript color... //[UIColor colorWithRed:0.23 green:0.50 blue:0.82 alpha:0.90];
```

?

**Comment by maknz** [comment]
I've released a v1.0.1 that will do exactly as you proposed. It simply returns the native implementation from the `show()` method.

e.g. 

``` js
var impl = loader.show();

if (page.ios) {
  // impl is an MBProgressHUD
  impl.dimBackground = true;
} else if (page.android) {
  // impl is a ProgressDialog
}
```

**Comment by NathanWalker** [comment]
Sweet! Thanks a million!!
